functions/package.json

```diff
@@ -1,26 +1,30 @@
 {
   "name": "shiritori-agent",
+  "version": "0.0.2",
   "description": "shiritori agent fullfillment",
-  "version": "0.0.1",
-  "private": true,
-  "license": "Apache Version 2.0",
   "author": "Google Inc.",
+  "license": "Apache License 2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/actions-on-google/actionssdk-shiritori-ja-nodejs"
+  },
+  "private": true,
   "engines": {
-    "node": "~6.0"
+    "node": "8"
```

Comment:
I see you're using Node 8, but not using any of `await` syntax in your index.js file.

Reply:
Are you referring to `functions/index.js` `shiritori/index.js` or both? :)

I wasn't sure if `async` `await` were supported with Node 8, but just checked with https://node.green/ that it seems to be the case.

Reply:
Done